cmake/protobuf-module.cmake.in

```diff
@@ -1,7 +1,4 @@
-if(PROTOBUF_SRC_ROOT_FOLDER)
-  message(AUTHOR_WARNING "Variable PROTOBUF_SRC_ROOT_FOLDER defined, but not"
-    " used in CONFIG mode")
-endif()
+# Functions
```

Comment:
'Simply include the module will not work'. Not true. I tried this locally and had some promising results.
'The module had a change over time.' This is exactly why I don't like this file.

By including the version of FindProtobuf.cmake that ships with the version of CMake the user has, we guarantee *by definition* that it will be compatible with that version of cmake. If the goal is legacy compatibility you can't get better than that. It also means we'd get any updates for free.